packages/frontend/src/components/select_sports.tsx

```diff
@@ -0,0 +1,47 @@
+import type { Sport } from "@esp-group-one/types";
+import { useState } from "react";
+import type { ChangeEvent } from "react";
+import { sportToColour } from "../../util/sport";
+
+interface SportListProps {
+  sports: Sport[];
+}
+
+export function SelectSport({
+  sports: sportsOptions,
+  currentSport,
+}: SportListProps & { currentSport: Sport }) {
```

Comment:
The `SportListProps` is only used here so you can just put `currentSport` into the interface above